app/Http/Controllers/AuthController.php

```diff
@@ -17,11 +18,19 @@ public function __construct(UserService $userService)
     }
     public function register(RegisterRequest $request)
     {
-        $user = $this->userService->store($request->validated());
+        $userRegister = $this->userService->store($request->validated());
 
-        $token = $user->createToken('API Token')->accessToken;
+        $token = $userRegister->createToken('API Token')->accessToken;
 
         return response()->json(['token' => $token], Response::HTTP_CREATED);
     }
+    public function login(LoginRequest $request)
+    {
+        $userLogin = $this->userService->login($request->validated());
```

Comment:
$userLogin sounds like a process but the variable contains the instance of the User class. So just name it as '$user'